cmd/popmd/README.md

```diff
@@ -0,0 +1,156 @@
+# Hemi PoP Miner (`popmd`)
+
+`popmd` is a **lightweight PoP (Proof-of-Proof) miner**, which wraps the `popm` service.
+
+![Depiction of L1 Profile](images/popminer.svg)
+
+`popmd` will periodically receive information about the current state of the hemi network (a keystone), constructs 
+a bitcoin transaction embedding the aforementioned keystone, and broadcasts these transactions - resulting in 
+rewards, once validated.
+
+## System Requirements
+
+`popmd` is a **lightweight daemon** and most systems should be able to run popmd over long periods of time without requiring significant resources, as it off-loads most heavy functionality to other daemons it connects to.
+
+Furthermore, a **GPU is neither required nor helpful** to the functioning of `popmd`.
+
+## Running `popmd`
+
+In order to run `popmd`, you may either use _docker_, download a pre-built binary, or build the tool from source.
+
+**NOTE:** URLs for both a [BTC Gozer](../../bitcoin/wallet/README.md) with indexed keystones (such as [`tbcd`]((../tbcd/README.md).)), as well as an [HVM-aware op-geth](https://github.com/hemilabs/op-geth) instance, are required in order to use `popmd`.
+
+### Downloading Release Binaries (Recommended)
+
+Pre-built binaries are available on the [Releases Page](https://github.com/hemilabs/heminetwork/releases).
+
+After extracting the archive that matches your system, start `popmd` by running:
+
+```shell
+/path/to/popmd
+```
+
+### Running Local Docker Image
+
+The `heminetwork` repository provides docker images that can be used to run `popmd` using `Docker`.
```

Comment:
heminetwork provides docker files, not docker images.  The images are on dockerhub.